src/Purestone/Card.hs

```diff
@@ -20,16 +20,39 @@ import Purestone.JSON
 -------------------------------------------------------------------------------
 
 -- | Represents cards in the game.
-data Card = Card {
-    -- | The name of the card.
-    cardName :: Text,
-    -- | The card's flavour text.
-    cardDescription :: Text,
-    -- | The list of paradigms for this card.
-    cardParadigms :: NonEmpty Paradigm,
-    -- | The card's program.
-    cardProgram :: Program
-} deriving (Eq, Show, Generic)
+data Card = CardProgram {
```

Comment:
I think in terms of formatting something like:
```haskell
data Card
    = CardProgram {
        -- fields
    }
    | CardScript {
```
is probably nicer

Reply:
Done with https://github.com/fpclass/perpetual-haskelling-initiative/pull/15/commits/ea621cf4c82a4a03a1dc29e4ae2dc551fe0ef230